stubs/uWSGI/uwsgidecorators.pyi

```diff
@@ -0,0 +1,180 @@
+from collections.abc import Callable
+from typing import Any, Generic, TypeVar, overload
+from typing_extensions import Literal, ParamSpec
+
+from uwsgi import _RPCCallable
+
+_T = TypeVar("_T")
+_T2 = TypeVar("_T2")
+_SR = TypeVar("_SR", bound=Literal[0, -1, -2] | None)
+_SignalCallback = TypeVar("_SignalCallback", bound=Callable[[int], Any])
```

Comment:
Suggest renaming this to `_SignalCallbackT` so it's clear that it's a TypeVar, not an alias.